asf_welsh_energy_consultation/getters/get_data.py

```diff
@@ -13,11 +13,16 @@
 
 from asf_core_data.getters.epc.data_batches import get_batch_path
 from asf_core_data.config import base_config
-from asf_core_data.getters.data_getters import download_core_data, logger
+from asf_core_data.getters.data_getters import (
+    download_core_data,
+    logger,
+    download_from_s3,
+)
 
 import pandas as pd
 import numpy as np
 import os
+import dask.dataframe as dd
```

Comment:
don't we have to add dask to the requirements then?

Reply:
Added ty